src/ape_geth/providers.py

```diff
@@ -215,20 +215,19 @@ def connect(self):
             if not self._web3.isConnected():
                 self._geth.disconnect()
                 raise ConnectionError("Unable to connect to locally running geth.")
+        elif "geth" in self.client_version.lower():
+            logger.info(f"Connecting to existing Geth node at '{self._clean_uri}'.")
```

Comment:
I am wondering if this could be reused in this way to simplify this if statement at all?
`logger.info(f"Connecting to existing '{self.client_version.lower()}' node at '{self._clean_uri}'.")`

Reply:
I messed around this. It is kind of tricky because of the `else:` statement where the client is unknown as well as the settings that get set for Nethermind and Erigon. However, we can share a helper method to enforce a consistent log